src/services/CLIService.ts

```diff
@@ -101,24 +112,46 @@ export class CLIService implements IService{
             userCompose,
             userComposeDir,
             blocklisting,
-            startup
+            startup,
+            verbose
         };
 
         return currentArgv;
     }
 
     public setCurrentArgv(argv: ArgumentsCamelCase<{}>): void {
-        this.currentArgv = argv;
+        const state = argv._[0] as string
+        this.currentArgv = {
+            ...argv,
+            detached: this.isStartup(state) ? argv.detached : true,
+        };
+    }
+
+    private isStartup(state: string): boolean {
+        switch (state) {
+            case 'stop':
+                return true;
+            case 'generate-accounts':
+                return true;
+            case 'debug':
+                return true;
+            case 'start':
+                return false;
+            case 'restart':
+                return false;
+            default:
+                return true;   
+        };
```

Comment:
Some of the values in this swtich statement need revising

Reply:
revised  Offline.